src/schema/Transfer/dataloaders/DFSP.ts

```diff
@@ -30,31 +30,20 @@ const findDfsps = async (ctx: Context, transferIds: string[], type: DFSPType) =>
     },
     select: {
       transferId: true,
-      participantCurrency: {
-        include: {
-          participant: true,
-        },
-      },
+      participant: true
```

Comment:
Switched to getting the participant from the relation rather than partcipantCurrency which can be null.